notebooks/community/managed_notebooks/predictive_maintainance/Predictive_maintenance_usecase.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        serving_container_image_uri="us-docker.pkg.dev/vertex-ai/prediction/xgboost-cpu.1-1:latest",
</pre><p>Consider using the latest XGBoost image available (currently 1.4): https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers#xgboost</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>resolved</p>